### DISCUSSION TITLE
MAS core installation - IBM Cloud Pak for Data

### DISCUSSION THREAD

**Discussion by ritchie9710** [nishi2go/mas-standalone-install#2] [Q&A]
Hi, can you help... do you have any 'how to' in terms of setting up the Cloud Pak for Data elements within the Code Ready Container for the  core installation? I've followed your linkedin guide up to that point and not able to get any further and the IBM documentation is at best difficult!

**Comment by nishi2go** [comment]
Hi. Unfortunately, I don't have enough resources to run the CP4D now. You can find the great documents which it provided by an IBM partner. Please check it out: https://tjmcmanus.github.io/IBMPartnerDemo/Data35.html

**Reply by ritchie9710** [reply]
Thanks but did you need to install CP4D as part of the mas-standalone-installation in Code Ready Containers?

**Reply by nishi2go** [reply]
Actually, CP4D is optional. You can run the Manage to create a database outside the OCP.
